app/assets/javascripts/components/story/StoryItem.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import classname from 'classnames';
+import PropTypes from 'prop-types';
+import {
+  classIconRule,
+  iconRule,
+  isStoryDontEstimated,
+  IsShow,
+  estimateRule,
+  labelSplit,
+  isRelease,
+} from '../../rules/story';
+
+const StoryPoints = () => (
+  <div>
+    <span className="Story__estimate">1</span>
+    <span className="Story__estimate">2</span>
+    <span className="Story__estimate">3</span>
+    <span className="Story__estimate">5</span>
+    <span className="Story__estimate">8</span>
+  </div>
+)
+
+const StateActions = ({ story_type, estimate }) => (
+  <div className='Story__actions'>
+    <IsShow logic={isStoryDontEstimated(story_type, estimate)}>
```

Comment:
I'd prefer to avoid these components that simulate an `if` in React because it loses the advantage of having JSX like pure JS. You could replace it with this:

```js
<div className='Story__actions'>
  {
    isStoryDontEstimated(story_type, estimate)
      ? <StoryPoints />
      : <ButtonStart />
  }
</div>
```

This way we avoid having to understand how a new component works (the name of the prop for the logic, ...) and have a JS ternary operator.

Reply:
I dont link to use ternary in JSX, In my opinion using ternary is very ugly, but i am ok

Reply:
Adding a new component to have this logic is even uglier hahaha, also, the ternary is pretty straightforward, the person who's using doesn't have to think "how do I make an else with that?" (different from having a `IsShow` component) because it's simply pure JS.